examples/ServoStop/ServoStop.ino

```diff
@@ -0,0 +1,34 @@
+#include <Servo.h>
+byte trigPin = 4;
+byte echoPin = 2;
+byte servoPin = 9;
+
+Servo myservo;  // create servo object to control a servo
+int pos = 0; // variable to store the servo position
+void setup() {
+  Serial.begin(9600);
+  pinMode(trigPin, OUTPUT);
+  pinMode(echoPin, INPUT);
+  myservo.attach(servoPin);  // attaches the Servo on pin 9 to the servo object
```

Comment:
```suggestion
  myservo.attach(servoPin);  // attaches the servo on pin 9 to the Servo object
```
The first mention of servo is about the servo motor. The second mention is about the Servo class.

Reply:
Comment Fixed - Please Review

Reply:
OK, but what about all the other changes I asked you to make?

Reply:
New Commit made - Sketch description and comments added.
Please Review